.github/workflows/golangci-lint.yaml

```
@@ -0,0 +1,25 @@
+name: golangci-lint
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
```

```suggestion
on: [push, pull_request]
```

Is there any reason this should not always always run?